packages/ui-common/src/components/RoomPlayer/PlayerMenu.tsx

```diff
@@ -108,9 +79,12 @@ function PlayerMenu({ quitRoom, players, curPlayer }: PlayerMenuProps): React.Re
             <IconButton
               size="small"
               sx={{ borderRadius: 1 }}
-              onClick={() => {
-                setAboutModalOpen(!aboutModalOpen);
-              }}
+              href="/"
+              onClick={() => logger.info(
+                `Platform link clicked from game: ${game?.id ?? 'NO ID FOUND'}`,
```

Comment:
This will just log to the console, not to our analytics. It may be worth looking into how to do this with our current firebase analytics setup. Its possible that this can be already done by default with keeping the referrer